frontend/src/app/pages/exercises/exercise/shared/trainer-toolbar/trainer-toolbar.component.html

```diff
@@ -9,6 +9,12 @@
     <button (click)="openTransferOverview()" class="btn btn-primary ms-3">
         Transferübersicht
     </button>
+    <button
+        (click)="openExerciseStatisticsModal()"
+        class="btn btn-primary ms-3"
+    >
+        Auswertung
```

Comment:
Maybe `Statistiken` instead?

Reply:
sounds better. "Auswertungen" implies some conclusions drawn by the software.